src/code42cli/cmds/cases.py

```diff
@@ -0,0 +1,246 @@
+from collections import OrderedDict
+
+import click
+from py42.exceptions import Py42BadRequestError
+from py42.exceptions import Py42NotFoundError
+
+from code42cli.click_ext.groups import OrderedGroup
+from code42cli.options import format_option
+from code42cli.options import sdk_options
+from code42cli.output_formats import OutputFormatter
+
+
+_HEADER_KEYS_MAP = OrderedDict()
+_HEADER_KEYS_MAP["name"] = "Name"
+_HEADER_KEYS_MAP["assignee"] = "Assignee"
+_HEADER_KEYS_MAP["status"] = "Status"
+_HEADER_KEYS_MAP["createdAt"] = "Creation Time"
+_HEADER_KEYS_MAP["findings"] = "Notes"
+
+name_option = click.option(
+    "--name", required=False, help="Name of the case.", default=None
+)
+assignee_option = click.option(
+    "--assignee", required=False, help="User UID of the assignee.", default=None
+)
+description_option = click.option(
+    "--description", required=False, help="Description of the case.", default=None
+)
+notes_option = click.option(
+    "--notes", required=False, help="Notes on the case.", default=None
+)
+subject_option = click.option(
+    "--subject", required=False, help="User UID of a subject of the case.", default=None
+)
+status_option = click.option(
```

Comment:
The status option should probably include `type=click.Choice(CaseStatus.choices()),` (after importing the enum)

Reply:
the tests are failing after making this change, locally executing `pytest` passes the tests whereas `tox` command fails.

Reply:
Why does it fail?

Reply:
it cannot load module `py42.clients.cases` from the statement `from py42.clients.cases import CaseStatus`

Reply:
As per my understanding, `tox` fetches released version of `py42` and hence cannot load the module, whereas since I have configured `py42` to point to local `py42` module, `pytest` is able to load the module.  So once `cases` is released in `py42` this might get fixed.

Reply:
Right, it will get fixed when we release py42